src/server/generic_family.cc

```diff
@@ -288,8 +288,8 @@ void Renamer::Find(Transaction* t) {
     auto& db_slice = EngineShard::tlocal()->db_slice();
     auto [it, exp_it] = db_slice.FindReadOnly(t->GetDbContext(), res->key);
 
-    res->found = IsValid(it);
```

Comment:
nit: you could provide `IsValid` override in db_slice.h and greatly reduce the size of the PR.

Reply:
btw, this is why it existed in the first place - I had different type of iterators in the past and wanted to handle them "cleanly".

Reply:
oh, i forgot about this comment. Please do not review yet, I'll fix this as well.